### ISSUE TITLE
Missed minor version bump?

### ISSUE THREAD

**Issue by Nashenas88** [dimforge/nalgebra#1328]
I ran into an issue where data I had serialized previously failed to deserialize. I was able to track it down to https://github.com/dimforge/nalgebra/compare/v0.32.2...v0.32.3#diff-627412b64dcb29bce666a665846074cc243329d6c336e70fcf9d04da018f6793R195 where the serialization of anything backed by array storage is now different. In my project I'm using postcard as the serialization format, and the two are encoded differently.

That seems like a big enough change in behavior to have needed a bump from 0.32.2 to 0.33.0 (as opposed to 0.32.3). Not sure if it's worth it now but thought I should bring it up in case you all could consider changing how you bump versions in the future. It took me a couple days to figure out what was going wrong.

**Comment by Ralith** [comment]
Yep, you're the second to report this. We should probably yank and bump.

**Comment by Andlon** [comment]
I agree with @Ralith here. @sebcrozet would you be willing to yank and publish a new minor version whenever you are able to find the time?
